lib/Mojo/Server.pm

```diff
@@ -21,6 +22,7 @@ sub build_app {
 sub build_tx {
   my $self = shift;
   my $tx   = $self->app->build_tx;
+  if (my $p = $self->trusted_proxies) { $tx->req->trusted_proxies($p) }
```

Comment:
Looks like this is always true.

Reply:
Yeah, I wondered about that last night. Should I then also check `@$p` or just assign it always?

Reply:
🤷

Reply:
Does `if (my $p = $self->trusted_proxies) { $tx->req->trusted_proxies($p) if @$p }` seem weird? It seems weird to me.

Reply:
Very weird.

Reply:
How about `push @{$tx->req->trusted_proxies}, @{$self->trusted_proxies};`? It has the advantage of shallow cloning (which we probably want anyway) and it avoids lots of tests.

Reply:
Also, since this happens after the build tx hook, it is possible they will have set something on the req object.

Reply:
I've gone with this approach.